src/request/components/request-detail-panel-execution.jsx

```diff
@@ -52,21 +54,54 @@ var getMessages = (function() {
     }
 })();
 
+var combineMongoMessages = function(message) {
```

Comment:
Unfortunately the client hasn't been converted to TS or have Babel in place yet. On the todo list.